### ISSUE TITLE
Windows crash when using negotiated data channel (2nd connection)

### ISSUE THREAD

**Issue by simonbengtsson** [flutter-webrtc/flutter-webrtc#903]
I'm developing a file sharing app and experienced that  the windows app was crashing every time a second file was received. After some debugging I found that it can easily be reproduced by creating a data channel on a second peer connection with a negotiated data channel (see example below). Although not optimal my workaround is to not use a negotiated data channel for now.

The only crash logs shown are below so not very helpful.

```
Lost connection to device.
Exited (sigterm)
```

**To Reproduce**
```dart
await createPeerConnection({});
var pc = await createPeerConnection({});

var init = RTCDataChannelInit();
init.negotiated = true;
await pc.createDataChannel('channel', init); // This crashes the windows app
```

**Platform information**

* **Flutter version**: 2.12.0-4.1.pre (channel beta)
* **Plugin version**: 0.8.5 (and older ones as well)
* **OS**: Windows 11

**Comment by simonbengtsson** [comment]
Likely fixed in 0.8.9.

**Comment by simonbengtsson** [comment]
Confirmed to be fixed with 0.8.9 👌
